src/lib.rs

```diff
@@ -160,6 +160,8 @@ impl Url {
 
     /// Returns whether or not the URL can be parsed or not.
     ///
+    /// For more information, read [WHATWG URL spec](https://url.spec.whatwg.org/#dom-url-canparse)
```

Comment:
You can also add an introduction at the top with `//!`

Reply:
Nice!